### DISCUSSION TITLE
Backpack CRUD preview shows error after composer update.

### DISCUSSION THREAD

**Discussion by Zannatul** [Laravel-Backpack/community-forum#244] [General] [answered]
Hi @tabacitu 
Greetings of the day. I am using your handicraft in daily use. But I face some problem after my last composer update. I use laravel 9. Before composer update it worked fine but after that it broke. It shows 

`array_map(): Argument #2 ($array) must be of type array, null given`
![show error](https://user-images.githubusercontent.com/5654915/198866211-8bc7d21e-006e-4041-a5e4-1f2a32618fe0.png)

Can you please help me about this? I am using pro version.

**Comment by tabacitu** [comment, marked as answer]
Sorry for the late reply. A "composer update" should fix that 💪
